api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -0,0 +1,112 @@
+package gov.cms.ab2d.api.controller;
+
+
+import gov.cms.ab2d.api.util.Constants;
+import io.swagger.annotations.*;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.NotBlank;
+import java.io.IOException;
+
+@Api(value = "Bulk Data Access API", description =
+        "API through which an authenticated and authorized PDP sponsor" +
+                " may request a bulk-data export from a server, receive status information " +
+                "regarding progress in the generation of the requested files, and retrieve these " +
+                "files")
+@RestController
+@RequestMapping(path = "/api/v1", produces = "application/json")
+/**
+ * The sole REST controller for AB2D's implementation of the FHIR Bulk Data API specification.
+ */
+public class BulkDataAccessAPI {
+
+    private static final String ALLOWABLE_OUTPUT_FORMATS =
+            "application/fhir+ndjson,application/ndjson,ndjson";
+
+    @ApiOperation(value = "Initiate Part A & B bulk claim export job")
+    @ApiImplicitParams(
+            @ApiImplicitParam(name = "Prefer", required = true, paramType = "header", value =
+                    "respond-async"))
+    @ApiResponses(
+            @ApiResponse(code = 202, message = "Export request has started", responseHeaders =
+            @ResponseHeader(name = "Content-Location", description = "URL to query job status",
+                    response = String.class))
+    )
+    @ResponseStatus(value = HttpStatus.ACCEPTED)
+    @GetMapping("/Patient/$export")
+    public ResponseEntity<Void> exportAllPatients(
+            @ApiParam(value = "String of comma-delimited FHIR resource types. Only resources of " +
+                    "the specified resource types(s) SHALL be included in the response.",
+                    allowableValues = "ExplanationOfBenefits")
+            @RequestParam(required = false, name = "_type") String resourceTypes,
+            @ApiParam(value = "A FHIR instant. Resources will be included in the response if " +
+                    "their state has changed after the supplied time.")
+            @RequestParam(required = false, name = "_since") String since,
+            @ApiParam(value = "The format for the requested bulk data files to be generated.",
+                    allowableValues = ALLOWABLE_OUTPUT_FORMATS, defaultValue = "application/fhir" +
+                    "+ndjson"
+            )
+            @RequestParam(required = false, name = "_outputFormat") String outputFormat) throws IOException {
+        //String encoded = FHIRUtil.outcomeToJSON(FHIRUtil.getSuccessfulOutcome("OK"));
```

Comment:
Does this encoded variable still need to be used or is it for something that will come later?

Reply:
Discussed on slack and this is something that will come later.

Reply:
yup, commented out for now to make PMD happy